src/user/system/CaptainManager.ts

```diff
@@ -637,8 +647,13 @@ class CaptainManager {
                             key: 'TELEGRAM_CHAT_ID',
                             value: netDataInfo.data.telegram.chatId,
                         })
+                        envVars.push({
+                            key: 'SEND_TELEGRAM',
+                            value: 'YES',
```

Comment:
These are redundant. They can be set to YES in the `run.sh` file of the netdata container if the parent env var (e.g.,  TELEGRAM_CHAT_ID) is present.

Reply:
Ok

Reply:
So with last changes, should be good now, give it a go.